docs/howto/mbed-tools-howto.md

```diff
@@ -126,13 +126,21 @@ Mbed Tools has two environment variables that you can set for a project:
     - `ONLINE`: Alway use the online database.
     - `OFFLINE`: Always use the offline database.
 
-To set values, create an `.env` file in the root directory of the project. The file should contain definitions in the `<VARIABLE>=<value>` format.
+These variables can be set either directly via environment variables
+or using a `.env` file containing the variable definitions as follows:
 
-For more information on overriding defaults, use
+`VARIABLE1=<value>`
```

Comment:
I'd recommend using triple backticks so this shows up like a file snippet.

```
VARIABLE1=<value>
VARIABLE2=<value>
```

instead of
`VARIABLE1=<value>`
`VARIABLE2=<value>`